networking/src/Parsers/QuestionTemplateParser.ts

```diff
@@ -41,7 +41,13 @@ export class QuestionTemplateParser {
       if (isNullOrUndefined(id) ||
           isNullOrUndefined(title) ||
           isNullOrUndefined(owner) ||
-          isNullOrUndefined(version)) {
+          isNullOrUndefined(version) ||
+          isNullOrUndefined(domain) ||
+          isNullOrUndefined(cluster) ||
+          isNullOrUndefined(grade) ||
+          isNullOrUndefined(standard) ||
```

Comment:
Are these all reqiured? I remember we discussed some CCSS fields might not be available for certain questions